tracer/src/tracer.rs

```diff
@@ -101,11 +102,14 @@ struct TraceEvent {
 
 pub fn trace_point_log(event: &'static str) {
     let trace_event = TraceEvent {
+        // SAFETY: access global variable
```

Comment:
I think the safety comments around `TRACER` need to be further clarified. @rbradford

Reply:
I updated this patch. I tried to reason about this module but admittedly I didn't look too deeply.